pages/osx/afplay.md

```diff
@@ -0,0 +1,19 @@
+# afplay
+
+> Command-line audio player.
+
+- Play a sound file (waits until playback ends):
+
+`afplay {{path/to/file}}`
+
+- Play a sound file at 2x speed:
+
+`afplay -r 2 {{path/to/file}}`
+
+- Play a sound file at half speed:
+
+`afplay -r 0.5 {{path/to/file}}`
+
+- Play the first N seconds of a sound file:
```

Comment:
Do you know what the "t" stands for? Maybe we could incorporate that into the description.

Reply:
Going by the usage text, it seems to stand for "time":

```
  {-t | --time} TIME
    play for TIME seconds
```

I don't know how I'd rewrite the description to include the word "time" without sounding (no pun intended) contorted. "Play a sound file for a maximum amount of time" maybe? "Limit playback time"? I wrote it this way because it seems like the most straightforward to understand.

(The current wording already omits one detail: the time limit is applied after the rate adjustment, so if you play at 2x speed and use `-t N` it will still play sound for N seconds, but get twice as far along into the file. I did not mention it in the description because I couldn't come up with a succinct enough wording to explain that.)

Reply:
I see what you mean -- it's a common struggle we face as we balance succinctness, correctness and beginner-friendliness. I actually like something like `Play a sound file for a limited time` or `Play a sound file up to the given timestamp (in seconds)`, but I agree the current wording is simpler. Another strategy that is often handy is using the spelled-out version of the option, so in this case `--time`. Do you think any of these could improve the example?

(And yeah, don't worry about the nuance of this option's interaction with the rate one.)

Reply:
Using `--time` and keeping the description as-is sounds good to me.

Reply:
For future reference, the expanded option name `--time` was added in [`cd068a4`](https://github.com/tldr-pages/tldr/pull/3403/commits/cd068a4b2cc341d3d602050e40dbec482956d4d0).